.github/workflows/image-factory.yaml

```
@@ -197,8 +191,14 @@ jobs:
 
       - name: build and push trillian
         run: |
-          cd rekor
-          KO_PREFIX=quay.io/securesign KO_DOCKER_REPO=quay.io/securesign KO_DEFAULTBASEIMAGE=registry.access.redhat.com/ubi9/ubi-micro make ko-trillian
+          cd trillian
+          mv redhat/overlays/log_server/Dockerfile .
+          docker build -t quay.io/securesign/trillian_log_server:${TRILLIAN_VER} .
+          docker push quay.io/securesign/trillian_log_server:${TRILLIAN_VER}
+          mv Dockerfile redhat/overlays/log_server
+          mv redhat/overlays/log_signer/Dockerfile .
+          docker build -t quay.io/securesign/trillian_log_signer:${TRILLIAN_VER} .
+          docker push quay.io/securesign/trillian_log_signer:${TRILLIAN_VER}
```

I think you could simplify this by providing the paths (and you should probably use podman here).
```suggestion
          podman build -t quay.io/securesign/trillian_log_server:${TRILLIAN_VER} ./redhat/overlays/log_server/Dockerfile
          podman push quay.io/securesign/trillian_log_server:${TRILLIAN_VER}
          podman build -t quay.io/securesign/trillian_log_signer:${TRILLIAN_VER} ./redhat/overlays/log_signer/Dockerfile
          podman push quay.io/securesign/trillian_log_signer:${TRILLIAN_VER}
```

---

So the rationale behind the docker files being moved in and out of the folder as simply because Dockerfiles are supposed to be run from root. Something similar can be seen on line 235 for the Trillian DB image build. I can fully configure them as above but it would require those Dockerfile pathings to be altered.

---

Oh you are right. The `PATH` parameter is meant to be the root of the build, not the path to the Dockerfile. My mistake. But please do use podman here.

---

Will do.